### PR TITLE
Add list_queues method to rabbitmq container

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

File: seaworthy/containers/provided.py [hunk 1]
```diff
@@ -167,6 +167,19 @@ def list_vhosts(self):
         """
         return output_lines(self.exec_rabbitmqctl('list_vhosts'))
 
+    def list_queues(self, vhost):
```
Comment (JayH5) [inline seaworthy/containers/provided.py]: Could we do `vhost=None` and then default to `self.vhost`? I see `list_policies` doesn't even let you specify a vhost :/

Comment (jerith) [reply seaworthy/containers/provided.py]: Good point.

**Comment by jerith** [comment]
I still need to write proper tests for this.

File: seaworthy/containers/provided.py [hunk 2]
```diff
@@ -191,3 +209,39 @@ def broker_url(self):
...
         return 'amqp://{}:{}@{}/{}'.format(
             self.user, self.password, self.name, self.vhost)
+
+    def _setup_management(self):
+        """
+        Enable the management API plugin and install curl.
+        """
+        if self._management_available:
+            return
+        self.inner().exec_run(['apk', 'add', '--update', 'curl'])
```
Comment (JayH5) [inline seaworthy/containers/provided.py]: You probably want `--no-cache` instead of `--update`

Comment (JayH5) [reply seaworthy/containers/provided.py]: Oh and there is BusyBox `wget`?

Comment (jerith) [reply seaworthy/containers/provided.py]: Busybox `wget` is very limited. Specifically, it doesn't let us set the HTTP method and thus we can't make `PUT` requests.

**Pull request merged**